testhelpers/org.eclipse.n4js.ui.tests.helper/src/org/eclipse/n4js/tests/hover/AbstractN4JSHoverTest.xtend

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2019 HAW Hamburg.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Philip Aguilar Bremer
+ * 	 Max Neuwirt
```

Comment:
Could you write here

- Philip Aguilar Bremer, Max Neuwirt; HAW Hamburg

This would then explain the copyright. Otherwise the lawyers cannot relate your names to the HAW copyright.